### ISSUE TITLE
Serilog integration

### ISSUE THREAD

**Issue by Allann** [mayuki/Cocona#110]
Question:

1. What is the easiest way to integrate Serilog (with appsetting configuration) into Cocona?  

2. How do I turn off the default logging into the console as it gets very confusing for the user as the commands use console as does the logging.

Thanks in advance.

**Comment by j-francisco** [comment]
I'm using NLog instead of Serilog, but I was able to stop the console logging with:

```
builder.Logging.ClearProviders();
builder.Logging.AddNLog();
```

**Comment by jtsai-osa** [comment]
It's pretty much the same as integrating with asp.net core applications. After adding the Serilog.AspNetCore nuget package you create a `CoconaAppBuilder` by calling `CoconaApp.CreateBuilder`. And with `CoconaAppBuilder` you either call `Host.UseSerilog` to use Serilog as the main log factory, or call `Logging.AddSerilog` to add Serilog as a log provider.

**Comment by alindgren** [comment]
@jtsai-osa shouldn't one use the Serilog.Extensions.Hosting nuget package instead of Serilog.AspNetCore? The readme for Serilog.Extensions.Hosting states Serilog.AspNetCore bundles Serilog.Extensions.Hosting package and includes other ASP.NET Core-specific features.

**Comment by jtsai-osa** [comment]
@alindgren Yup you are absolutely correct. I'm using Serilog.AspNetCore because I need the sinks and being able to configure the log from appsetting.json. Your answer is definitely more appropriate since Serilog.Extensions.Hosting is the actual package that allows wiring up Serilog with Cocona. Thanks for the correction.
